rust/src/error.rs

```diff
@@ -27,6 +26,21 @@ pub enum Error {
     Utf8(Utf8Error),
 }
 
+impl Debug for Error {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::NotFound => write!(
+                f,
+                "Shared library file 'libyamlscript.so.{0}' not found
+Try: curl -sSL yamlscript.org/install | VERSION={0} LIB=1 bash
+See: https://github.com/yaml/yamlscript/wiki/Installing-YAMLScript",
+                &super::LIBYAMLSCRIPT_FILENAME["libyamlscript.so.".len()..]
```

Comment:
I would look to the python code for guidance:
https://github.com/yaml/yamlscript/blob/main/python/lib/yamlscript/__init__.py#L34-L40 

The installer runs `make install` in the release tarball dir.
But yes the macOS release tarballs for libyamlscript contain a `.dylib` file not a `.so` file:
https://github.com/yaml/yamlscript/releases

The version bumps are controlled by https://github.com/yaml/yamlscript/blob/main/.version.sh
Something like this would be fine:
```
const LIBYAMLSCRIPT_VERSION: &str = "0.1.49";
```
https://github.com/yaml/yamlscript/blob/main/.version.sh#L28 is the current rust bump but needs to be fixed...

Reply:
Any updates on the dylib?

Reply:
Rust supports detecting both "linux" and "unix" systems. I don't remember exactly in which category each OS is. The binding will currently work with "linux" systems and "macos" systems. If a system is unix but not linux, this will not work.